<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	background-color: white;
	color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_TaSL">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_TaCC">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TABLE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_CC">
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>CHAT</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>CONSOLE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>SEVER
LIST</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_Join"/>
    <addaction name="action_Disconnect"/>
    <addaction name="action_Rejoin"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>Изменить</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="View">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_Table"/>
    <addaction name="action_Chat"/>
    <addaction name="action_Console"/>
    <addaction name="action_ServerList"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Join">
   <property name="text">
    <string>Присоединиться</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="text">
    <string>Отключиться</string>
   </property>
  </action>
  <action name="action_Rejoin">
   <property name="text">
    <string>Перезайти</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_Table">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Стол</string>
   </property>
  </action>
  <action name="action_Chat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Чат</string>
   </property>
  </action>
  <action name="action_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Консоль</string>
   </property>
  </action>
  <action name="action_ServerList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Список Серверов</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
